_posts/2021-02-23-OpenSSL.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: Switching back to OpenSSL
+layout: post
+---
+
+The Void Linux team is planning on [performing the switch to
+OpenSSL](https://github.com/void-linux/void-packages/pull/21056) before March
+is over.
+
+For most users, this shouldn't be noticeable, since we will be pushing the same
```

Comment:
there should be no noticeable change, with the exception of a small handful of packages all your existing software will continue to work.  If your system has explicit dependencies on LibreSSL, you will of course need to take action to ensure your system continues to function after the switch.